src/py/workflows/nextstrain_run/export.py

```diff
@@ -0,0 +1,243 @@
+import csv
+import io
+import json
+from pathlib import Path
+from typing import Any, Mapping, MutableMapping, Set, Tuple
+
+import click
+from sqlalchemy.orm import aliased, joinedload, with_polymorphic
+
+from aspen.config.config import RemoteDatabaseConfig
+from aspen.database.connection import (
+    get_db_uri,
+    init_db,
+    session_scope,
+    SqlAlchemyInterface,
+)
+from aspen.database.models import (
+    Accession,
+    AlignedGisaidDump,
+    CalledPathogenGenome,
+    Entity,
+    HostFilteredSequencingReadsCollection,
+    PathogenGenome,
+    PhyloRun,
+    PublicRepositoryType,
+    Sample,
+    SequencingReadsCollection,
+    UploadedPathogenGenome,
+)
+from aspen.database.models.workflow import Workflow
+
+METADATA_CSV_FIELDS = [
+    "strain",
+    "virus",
+    "gisaid_epi_isl",
+    "genbank_accession",
+    "date",
+    "region",
+    "country",
+    "division",
+    "location",
+    "region_exposure",
+    "country_exposure",
+    "division_exposure",
+    "segment",
+    "length",
+    "host",
+    "age",
+    "sex",
+    "pango_lineage",
+    "GISAID_clade",
+    "originating_lab",
+    "submitting_lab",
+    "authors",
+    "url",
+    "title",
+    "paper_url",
+    "date_submitted",
+]
+
+
+@click.command("save")
+@click.option("--phylo-run-id", type=int, required=True)
+@click.option("sequences_fh", "--sequences", type=click.File("w"), required=True)
+@click.option("metadata_fh", "--metadata", type=click.File("w"), required=True)
+@click.option("builds_file_fh", "--builds-file", type=click.File("w"), required=True)
+def cli(
+    phylo_run_id: int,
+    sequences_fh: io.TextIOBase,
+    metadata_fh: io.TextIOBase,
+    builds_file_fh: io.TextIOBase,
+):
+    interface: SqlAlchemyInterface = init_db(get_db_uri(RemoteDatabaseConfig()))
+
+    with session_scope(interface) as session:
+        phylo_run_inputs = with_polymorphic(
+            Entity,
+            [PathogenGenome, AlignedGisaidDump],
+            flat=True,
+        )
```

Comment:
can you add a comment for what this should be returning? Is it supposed to be returning the pipeline parameters passed into ncov?

Reply:
Are you referencing the `with_polymorphic` call?

Reply:
I updated with an explanation of what the `with_polymorphic` call does.  Let me know if you meant something else.

Reply:
yeah i was confused about with_polymorphic, thanks for clarifying